src/main/java/com/library/java_library_management/controller/Controller.java

```diff
@@ -0,0 +1,52 @@
+package com.library.java_library_management.controller;
+
+import com.library.java_library_management.repository.TestModeRepository;
+import com.library.java_library_management.repository.Repository;
+import com.library.java_library_management.service.Service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Controller {
+    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
```suggestion
    private final BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
```

인스턴스 변수 선언 시 접근 제한자를 private로 선언하고, 초기화까지 한번에 이루어지는 경우 final을 선언하는 것을 권장합니다.